src/View/ViewableData.php

```diff
@@ -346,33 +298,38 @@ public function customise($data)
      *
      * This method should be overridden in subclasses to provide more context about the classes state. For example, a
      * {@link DataObject} class could return false when it is deleted from the database
-     *
-     * @return bool
      */
-    public function exists()
+    public function exists(): bool
     {
         return true;
     }
 
     /**
-     * @return string the class name
+     * Output this model in a format that is ready to be rendered directly into the view layer.
      */
-    public function __toString()
+    public function __toString(): string
     {
-        return static::class;
+        return (string) $this->forTemplate();
     }
```

Comment:
Found that this was necessary when doing the template POC - can remove from this PR and add to the refactor PR if there's any doubts about it

Reply:
On second thought I'll do this as follow-up work